### ISSUE TITLE
Shamir recovery: show which identifier were already entered

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-firmware#290]
It would be nice to display which shares (i.e. first four words) were already entered. If someone has 16 shares, it'd be quite useful.

Although this would be nice, I have no idea how to fit it on the screen. We would probably need a button on the "persistance homepage", which would take us to another screen with a scroll page of words?

I'm not sure it isn't too much of an effort. A simpler solution is to just inform the user after entering fourth word that the share was already entered (see #279), but that of course is not as good, because they do now know which one they haven't entered already.

Reported by @andrewkozlik. Depends on #270.

**Comment by tsusanka** [comment]
UX inclines to not implemeting this before data proves otherwise.

**Comment by tsusanka** [comment]
@ZdenekSL didn't we decide that we won't do this unless Product decides so?

**Comment by ZdenekSL** [comment]
Closed after discussion
